packages/jbrowse-cli/README.md

```diff
@@ -155,6 +183,27 @@ OPTIONS
 
 _See code: [@oclif/plugin-help](https://github.com/oclif/plugin-help/blob/v3.1.0/src/commands/help.ts)_
 
+## `jbrowse upgrade [LOCALPATH]`
+
+Upgrades JBrowse 2 to latest version
+
+```
+USAGE
+  $ jbrowse upgrade [LOCALPATH]
+
+ARGUMENTS
+  LOCALPATH  Location where JBrowse 2 is installed
+
+OPTIONS
+  -h, --help           show CLI help
+  -l, --latestVersion  Displays the latest JBrowse 2 version
```

Comment:
I think we might want to be able to upgrade to a custom version, too, e.g. upgrade from `1.1.1` to `1.1.5` when the latest is actually `1.1.7`.